data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,730 @@
+			`(To release the planet from tribute, select it from orbit and use the hail key.)`
+				accept
+
+event "slaver encounter"
+
+
+
+mission "Pirate Intro: Darkstone Slavers (Free)"
+	deadline 3
+	invisible
+	landing
+	source "Darkstone"
+	destination "Greenrock"
+	to offer
+		has "event: slaver encounter"
+		not "tribute: Darkstone"
+	npc kill
+		personality nemesis
+		government "Pirate"
+		fleet "Small Southern Pirates" 4
+	on offer
+		conversation
+			`Darkstone looks somewhat less depressing on your next landing. The village near the landing pad, rather than being blacked-out, is well-lit. Festive lanterns hang from each building, and there's a crackling bonfire in the center of town to beat back the chill. In fact, the village is far more lively than you'd have expected - the old folk and children must have come out of hiding, and even the adults are more numerous than last count. From what you can see during landing, everybody is assembled in the center of town - and your ship isn't the only one on the landing pad. Visitors?`
+			choice
+				`	(Disembark and head to the village center.)`
+			`	The outskirts are abandoned. There's not a single hand tending the fields, or in the outlying buildings. They've all assembled at the town center.`
+			`	As it turns out, they're not there to celebrate as you might have thought. No, the villagers are all being held at gunpoint by a gang of toughs in the center of the village green while a nasty-looking piece of work examines them one at a time, then directs them into one queue or another. Colonists in one are shackled together. Those in the other are made to lie flat with their hands behind their heads. Not a single colonist dares to look up; the spirit of defeat hangs too heavily over these broken souls.`
+			choice
+				`	"What's going on here?"`
+				`	(Stroll into the middle of town. Lock eyes with the leader of the slavers. Assume a gunfighter's stance.)`
+					goto duel
+				`	(Return to your ship. This is none of your business.)`
+					goto retreat
+				`	(Return to your ship... and start strafing the slavers.)`
+					goto attack
+			`	"No business of yours, friend. We know who you are, and we thank you for running off the last band of lowlifes that laid claim to these bodies. Now, if you'll excuse, we've got work to do. Piss off." The slaver chief roundly ignores any further attempts to communicate, but he at least seems to be in a good enough mood to not have you shot on the spot.`
+			choice
+				`	"These people are my responsibility, and I'll fight you for their freedom."`
+				`	(Return to your ship. This is none of your business.)`
+					goto retreat
+				`	(Return to your ship... and start strafing the slavers.)`
+					goto attack	
```

Comment:
```suggestion
				`	(Return to your ship... and start strafing the slavers.)`
					to display
						"armament deterrence" > 0
					goto attack
```

Reply:
That will prevent the player from "strafing the slavers" if they have to ships with weapons.